backend/service/chaos/experimentation/experimentstore/storer.go

```diff
@@ -292,11 +297,29 @@ func (s *storer) getExperiment(ctx context.Context, runId string) (*Experiment,
 	return &Experiment{Run: &run, Config: &config}, nil
 }
 
+func (s *storer) GetRuntimeGenerator(configTypeUrl string) (*RuntimeGeneration, error) {
```

Comment:
I don't see a connection between storer and runtimegenerator. Maybe put it elsewhere as a part of rtds_runtime_generator or something?